<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
color: black;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>441</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="logo">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>LIBRARY MANAGEMENT SYSTEM</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QStackedWidget" name="login_widget">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="page_admin">
        <widget class="QFrame" name="frame">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>50</y>
           <width>401</width>
           <height>241</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>4</number>
         </property>
         <widget class="QPushButton" name="loginBtn">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>170</y>
            <width>171</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 2px solid black;
color: rgb(0, 0, 0);
padding: 0 10px 0 10px ;
</string>
          </property>
          <property name="text">
           <string>Log in</string>
          </property>
         </widget>
         <widget class="QLabel" name="text_2">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>20</y>
            <width>101</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Admin ID</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="input_id">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>50</y>
            <width>361</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 2px solid black;
color: rgb(0, 0, 0);
padding: 0 10px 0 10px ;
</string>
          </property>
          <property name="maxLength">
           <number>6</number>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Enter your ID - 6 numbers</string>
          </property>
         </widget>
         <widget class="QLabel" name="text_3">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>90</y>
            <width>111</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="input_pass">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>120</y>
            <width>361</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 2px solid black;
color: rgb(0, 0, 0);
padding: 0 10px 0 10px ;
</string>
          </property>
          <property name="maxLength">
           <number>20</number>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Enter your password</string>
          </property>
         </widget>
         <widget class="QPushButton" name="backBtnLogIn">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>170</y>
            <width>171</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 2px solid black;
color: rgb(0, 0, 0);
padding: 0 10px 0 10px ;
</string>
          </property>
          <property name="text">
           <string>Back</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="page_welcome">
        <widget class="QLabel" name="text">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>70</y>
           <width>95</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>You are ?</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
        </widget>
        <widget class="QPushButton" name="adminBtn">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>120</y>
           <width>151</width>
           <height>41</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 2px solid black;
color: rgb(0, 0, 0);
padding: 0 10px 0 10px ;
</string>
         </property>
         <property name="text">
          <string>Admin</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
        <widget class="QPushButton" name="guestBtn">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>120</y>
           <width>151</width>
           <height>41</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 2px solid black;
color: rgb(0, 0, 0);
padding: 0 10px 0 10px ;
</string>
         </property>
         <property name="text">
          <string>Guest</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </widget>
       <widget class="QWidget" name="page_guest">
        <widget class="QFrame" name="frame_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>50</y>
           <width>401</width>
           <height>241</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>4</number>
         </property>
         <widget class="QLabel" name="text_8">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>40</y>
            <width>359</width>
            <height>27</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Full Name</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="input_name">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>100</y>
            <width>351</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 2px solid black;
color: rgb(0, 0, 0);
padding: 0 10px 0 10px ;
</string>
          </property>
          <property name="maxLength">
           <number>30</number>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Enter your name </string>
          </property>
         </widget>
         <widget class="QPushButton" name="backBtn">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>160</y>
            <width>111</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 2px solid black;
color: rgb(0, 0, 0);
padding: 0 10px 0 10px ;
</string>
          </property>
          <property name="text">
           <string>Back</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="enterBtn">
          <property name="geometry">
           <rect>
            <x>230</x>
            <y>160</y>
            <width>111</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 2px solid black;
color: rgb(0, 0, 0);
padding: 0 10px 0 10px ;
</string>
          </property>
          <property name="text">
           <string>Enter</string>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
    <zorder>login_widget</zorder>
    <zorder>logo</zorder>
    <zorder>horizontalSpacer</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>36</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuVu_Thi_Minh_Quy_20227257">
    <property name="title">
     <string>viciousircle</string>
    </property>
    <addaction name="actionVu_Thi_Minh_Quy"/>
    <addaction name="action20227257"/>
   </widget>
   <addaction name="menuVu_Thi_Minh_Quy_20227257"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action20227257">
   <property name="text">
    <string>20227257</string>
   </property>
  </action>
  <action name="actionVu_Thi_Minh_Quy">
   <property name="text">
    <string>Vu Thi Minh Quy</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
